### ISSUE TITLE
Web-api giving exception on clicking Dashboard in Blazor frontend

### ISSUE THREAD

**Issue by Sarmadjavediqbal** [fullstackhero/blazor-wasm-boilerplate#265]
![Screenshot 2023-04-28 165925](https://user-images.githubusercontent.com/46562878/235142535-6ee8ac74-4dff-491c-a1b0-15cf6fac7e22.png)

![Screenshot 2023-04-28 165800](https://user-images.githubusercontent.com/46562878/235142747-f2b850fc-904e-4348-8579-628e7f153d5f.png)

Web-api is throwing exception after clicking Dashboard in frontend (blazor-wasm).

**Comment by Sarmadjavediqbal** [comment]
any solutions??? s’il vous plaît

**Comment by MikaelHild** [comment]
Not sure, but it could be related to this:

https://github.com/jbogard/MediatR/issues/701

**Comment by Sarmadjavediqbal** [comment]
I found the solution. The definition of the Interface (IPipelineBehavior<TRequest, TResponse>) is:

Task<TResponse> Handle(TRequest request, CancellationToken cancellationToken, RequestHandlerDelegate<TResponse> next);

where as the class ValidationBehavior was implementing the above definition in wrong sequence as below:

Task<TResponse> Handle(TRequest request, RequestHandlerDelegate<TResponse> next, CancellationToken cancellationToken)

The definitions in both lines look the same but work differently. So I switched these two and it works just fine. Thanks @MikaelHild again for guiding.

It took a while but a little focus on the little details can save you hours of research.
